includes/api/route/AbstractRoute.php

```diff
@@ -99,4 +99,28 @@ public function register_routes() {
 			);
 		}
 	}
+
+	/**
+	 * Get remote data from an API request.
+	 *
+	 * @since NEXT
+	 *
+	 * @param  string $request The requested datapoint.
+	 *
+	 * @return array
+	 */
+	public function get_data( $request ) {
```

Comment:
@phatsk Just a heads up, I moved the original get_remote_data method back into the Press_Sync API class itself, since it seemed to be related to the get_remote_url and get_remote_response method. I don't really think of those methods as part of the API class, since most of that seems more focused on the syncing functionality, so we'll probably want to reconcile all of those methods into something that's more specific to the REST API. Whether that means it goes into an AbstractRoute, I'm not sure, but I think it's something worth talking about. This code change wouldn't have an impact on how my branch works, so we would be able to merge, but we'll probably want to figure out a more permanent solution.

Also, as a side note, when trying to retrieve a response for taxonomies when calling this method from the CLI, I wasn't seeing anything that showed ['body']['success'] - all of my requests were returning data in a different index. We might want to double-check how requests are built and what our expectations are for responses.